main.tf

```diff
@@ -6,6 +6,8 @@ locals {
 
   custom_data = var.custom_data != null ? base64encode(var.custom_data) : null
 
+  identity_type = join(", ", compact([var.system_assigned_identity_enabled ? "SystemAssigned" : "", length(var.identity_ids) > 0 ? "UserAssigned" : ""]))
```

Comment:
Using join makes you do the command in a single line, but the reason I did it in multiple steps was for readability. But maybe that is not an issue.

Reply:
Done ✅